code/_onclick/hud/hud.dm

```diff
@@ -80,7 +82,6 @@
 	mymob.healths = null
 	mymob.healthdoll = null
 	mymob.pullin = null
```

Comment:
these four above also need the same treatment, but I'm not going to bother with that in this PR.